release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -143,6 +143,13 @@ The Metering Operator is deprecated as of {product-title} 4.6, and is scheduled
 [id="ocp-4-9-scale"]
 === Scale
 
+[id="ocp-4-9-scale-memory-manager-in-beta-version"]
+==== Memory Manager feature in beta version
```

Comment:
It might be that the developers used the terms alpha and beta.  Product documentation indicates that software is generally available or technology preview.

Here's an [example](https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-storage-gcp-pd-csi-ga) that is clear about the software history regarding TP and GA.

If this RN entry is about the API versioning, such as this [example for an API group](https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-nodes-descheduler-apigroup), then please clarify that this update relates to an API.

Lastly, if there is any related information about working with the feature, the topology manager policies, or what the two policies accomplish, please add an xref to more information.

Reply:
Thanks for the quick review, Mike!
I don't think this is about the API versioning, but refers to the feature being in technology preview. @cynepco3hahue Could you confirm, please?
Also, is this a feature that should be added to the following table: https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-technology-preview ?

Reply:
@amolnar-rh, if it's technology preview, then I believe it should be added to the table--that's based on info from the OCP docs manual.